### ISSUE TITLE
[java] False positive on DontImportJavaLang when importing java.lang.ProcessBuilder.

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd#338]
**Rule Set:** java-imports/DontImportJavaLang

**Description:**

**Code Sample demonstrating the issue:**

```
import java.lang.ProcessBuilder.*;

public class Foo {
  private Redirect r;
}
```

**Running PMD through:** *Any*
